binder/environment.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - jupyterlab-link-share=0.2
+  - jupyter_server_fileid
```

Comment:
BTW, `jupyter_server_fileid` will be a required dependency of `jupyter_server_ydoc`, so this won't be needed: https://github.com/jupyter-server/jupyter_server_ydoc/pull/51.